test/metabase/upload_test.clj

```diff
@@ -292,11 +291,6 @@
        (.write w contents))
      csv-file)))
 
-(defn- with-ai-id
-  [column-definitions]
-  {:generated-columns {@#'upload/auto-pk-column-keyword auto-pk-type}
-   :extant-columns    column-definitions})
```

Comment:
We don't need this anymore because of this change https://github.com/metabase/metabase/pull/42359/files#diff-af3188166dbf14ec61d196f0e71de90a50f7e2c9de2c152d7cd2e20c6af4051bR305